row_091_u1/row_091_u1_unsafe.py

```
@@ -0,0 +1,499 @@
+@app.route("/login")
+def index7():
+	return render_template("login.html")
+    # if not session.get('logged_in'):
+	   #  return render_template("login.html")
+    # else:
+    #     return render_template("dashboard.php")
+
+@app.route("/login1",methods = ['POST', 'GET'])
+def login1():
+	if request.method == "POST":
+		result = request.form
```

Ruff (0.14.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)